Project.toml

```diff
@@ -3,5 +3,5 @@ Franklin = "713c75ef-9fc9-4b05-94a9-213340da978e"
 NodeJS = "2bd173c7-0d6d-553b-b6af-13a54713934c"
 
 [compat]
-Franklin = "0.10.28"
-NodeJS = "1.1.1"
+Franklin = "0.10"
+NodeJS = "1.1"
```

Comment:
Why this change?

Reply:
old habit, I usually put minor versions in compat fields, but I can revert